--- xf86-input-mouse-1.9.0/src/mouse.c.orig	2014-03-14 15:52:33.807027550 +0100
+++ xf86-input-mouse-1.9.0/src/mouse.c	2014-03-14 15:53:25.702328691 +0100
@@ -2043,7 +2043,7 @@ Emulate3ButtonsSoft(InputInfoPtr pInfo)
 {
     MouseDevPtr pMse = pInfo->private;
 
-    if (!pMse->emulate3ButtonsSoft)
+    if (pMse->emulate3Buttons || !pMse->emulate3ButtonsSoft)
         return TRUE;
 
     LogMessageVerbSigSafe(X_INFO, 4, "mouse: 3rd Button detected: disabling emulate3Button\n");
