diff -Naur xf86-input-mouse-1.9.1/src/mouse.c xf86-input-mouse-1.9.1.tpg/src/mouse.c
--- xf86-input-mouse-1.9.1/src/mouse.c	2014-08-12 06:36:34.000000000 +0000
+++ xf86-input-mouse-1.9.1.tpg/src/mouse.c	2014-12-14 16:30:34.062619882 +0000
@@ -2076,7 +2076,7 @@
 {
     MouseDevPtr pMse = pInfo->private;
 
-    if (!pMse->emulate3ButtonsSoft)
+    if (pMse->emulate3Buttons || !pMse->emulate3ButtonsSoft)
         return TRUE;
 
 #if defined(__NetBSD__) && defined(WSCONS_SUPPORT)
